code/items/equipments/entities/C4Entity.cs

```diff
@@ -149,4 +300,18 @@ public enum C4State
         Armed, //Shows wire cutting minigame
         Disarmed //Shows completed wire cutting minigame and "DISARMED" text. 
     }
+
+    public enum C4TimerPreset
+    {
+        None = -1,
+        Short,
+        Medium,
+        Long
+    }
+
+    public struct C4Preset
+    {
+        public int timer;
+        public int wires;
```

Comment:
capitalize pls

Reply:
They're not properties though

Reply:
but they are public vars.

Reply:
Interesting, it seems I've been following improper formatting for these the entire time.